grails-app/i18n/messages.properties

```diff
@@ -4096,7 +4096,7 @@ react.stocklists.fetch.fail.label=Unable to fetch stock lists
 
 # Products list
 react.productsList.header.label=Product List
-react.productsList.includeSubcategories.label=Include all products in all subcategories
+react.productsList.includeSubcategories.label=Include all subcategories
```

Comment:
I think we could get away with just "Include children" or "Include subcategories" since "all" is implied by the fact that we don't have more options. This is minor but important since the less words the user sees the less they need to process. 

We might need to do some more usability around this checkbox / label because the behavior that I expected would be "include subcategories" by default. Right now we exclude subcategories by default.
![image](https://user-images.githubusercontent.com/1162935/222579602-97b510e9-f84c-4e20-aac5-4648fc962463.png)

And the user is required to check the box in order to see them. 

So perhaps we really want to include the subcategories by default and allow the user to "Exclude subcategories" if they only want to see the products in the current category. 

Unfortunately, there's probably enough nuance between "Include" and "Exclude" in terms of the processing time required by the user (i.e. to calculate what products will be returned) so it would be better if this was researched a little more extensively.

Reply:
I remembered mint.com doing something around this but so far have only seen it with respect to their NLP-ish search query feature. 

![image](https://user-images.githubusercontent.com/1162935/222581398-e609efd1-64d1-4596-99b1-f354e30ac773.png)

NOTE: This is a screenshot from Google. I have way less than 5890 in my account.

Reply:
I found what I was looking for ... the first two options represent (parent and children vs just parent). 

![image](https://user-images.githubusercontent.com/1162935/222583564-4b34fd83-9911-4c07-8216-cb024130d377.png)

Reply:
> I think we could get away with just "Include children" or "Include subcategories" since "all" is implied by the fact that we don't have more options. This is minor but important since the less words the user sees the less they need to process.

You are right, "all" is probably unnecessary, I'll change it to _Include subcategories_